tests/build/chains.go

```diff
@@ -121,7 +125,25 @@ var _ = framework.ChainsSuiteDescribe("Tekton Chains E2E tests", Label("ec", "HA
 			// At a bare minimum, each spec within this context relies on the existence of
 			// an image that has been signed by Tekton Chains. Trigger a demo task to fulfill
 			// this purpose.
-			pr, err := kubeController.RunPipeline(tekton.BuildahDemo{Image: image, Bundle: fwk.TektonController.Bundles.BuildTemplatesBundle}, pipelineRunTimeout)
+
+			// Use docker-build pipeline and get it from the pipeline selector config
+			namespacedName := types.NamespacedName{
+				Name:      "build-pipeline-selector",
+				Namespace: "build-service",
+			}
+			bundle := ""
+			pipelineSelector := &buildservice.BuildPipelineSelector{}
+			Expect(fwk.CommonController.K8sClient.KubeRest().Get(context.TODO(), namespacedName, pipelineSelector)).To(Succeed())
+			for _, selector := range pipelineSelector.Spec.Selectors {
+				if selector.PipelineRef.Name == "docker-build" {
+					bundle = selector.PipelineRef.Bundle
+					break
+				}
+			}
+			// Can't continue without a docker-build pipeline
+			Expect(bundle).NotTo(Equal(""))
```

Comment:
would it make sense to move this part to `utils/tekton/controller.go`, keep the `Bundles` struct and extend it with other bundles [defined in pipeline selector CR](https://github.com/redhat-appstudio/infra-deployments/blob/b22275b78c0ac0c2ece2f981c25f86792752af47/components/build-service/build-pipeline-selector.yaml#L9-L27) (i.e. fbcBundle, dockerBuildBundle, etc.)?